### ISSUE TITLE
Optical Figures

### ISSUE THREAD

**Issue by Jordan-Dennis** [LouisDesdoigts/dLux#113]
Hi all,
Inspired by `numpyro` and `ticktack` drawing automatic figures, I was thinking that we could do something similar in `dLux` and draw the optical system using `pytikz`. The following image is the sort of thing that I would be after. 
![image](https://user-images.githubusercontent.com/93350879/190534334-67e05f29-76ca-412d-abd8-6149ec072ae2.png)
Let me know what you think,

Regards

Jordan

**Comment by benjaminpope** [comment]
Yep, absolutely. Similar to `ticktack` rendering the model. But it will be tricky:
- need to merge stacked aperture elements,
- include phase and amplitude,
- show Fresnel vs Fourier propagation

etc...
